inference/safety/main.py

```diff
@@ -0,0 +1,65 @@
+# A FastAPI server to run the safety pipeline
+
+import fastapi
+import uvicorn
+from blade2blade import Blade2Blade
+from fastapi.middleware.cors import CORSMiddleware
+from loguru import logger
+from oasst_shared.schemas import inference
+from settings import settings
+
+app = fastapi.FastAPI()
+
+
+# Allow CORS
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
```

Comment:
this is probably not needed, since no browser will try to reach this directly, right?

Reply:
Yeah I think you're right, removed